components/Blog/SideNav.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import './blog.scss'
+
+function SideNav ({ blogCountries, selectedCountry, onCountrySelected }) {
+  return (
+    <div className='side-nav-blog'>
+      {
+        blogCountries.map((country) =>
+          <a key={country} data-country={country} onClick={(event) => onCountrySelected(event.target.getAttribute('data-country'))}>
```

Comment:
The content in the sidebar is not accessible. The text is too light and fails to meet color contrast requirements. Even without vision issues, I can barely make out the text. Test your colors using this [color contrast checking tool](https://webaim.org/resources/contrastchecker/) to make sure they pass at least WCAG AA.

Since you are using an anchor (`a`) without an `href`, this is also not accessible. I cannot tab to it using a keyboard. All interactive elements need to be keyboard accessible. Also, semantically, links take you somewhere. This is not the best implementation of a filtering content, but I am struggling to find a good example of creating this in an accessible way. Until that can be done, I suggest we exclude the filtering element and just display all blog posts, since the sidebar will be inaccessible to may users.

Reply:
hey @meowwwls It would be great if you could comment your color as changes... I can test without any hassle :)

Reply:
For the color, I would try `#656565`. This has a good contrast ratio when paired with the current background-color.

![image](https://user-images.githubusercontent.com/16426195/66609733-6fc3fd00-eb7f-11e9-963d-29d001ddb6f9.png)

Reply:
thanks... for helping out...
<img width="203" alt="Screenshot 2019-10-11 at 3 34 07 AM" src="https://user-images.githubusercontent.com/17010408/66609893-cceb9d00-ebd7-11e9-9483-11bf6cf5c470.png">
currently I'm highlighting the selected text... 
I would be great if you give me the background-colour, text-colour (selected and non-selected)

Reply:
When you say selected text, do you mean when using `::selection` selector in your CSS? If so, I would just reverse it. So I would do:

un-selected:
```
color: #656565;
background-color: #f9f9f9;
```

selected:
```
color: #f9f9f9;
background-color: #656565;
```

Reply:
Its not the `psudo-elemnt` on user click we would render blogs related to it... So the school which user selects should be highlighted

Reply:
I added your color schema... I hope now it's fine... FYI
<img width="202" alt="Screenshot 2019-10-11 at 4 00 08 AM" src="https://user-images.githubusercontent.com/17010408/66611227-78e2b780-ebdb-11e9-9b51-c0d634003322.png">

Reply:
Thank you for clarifying! 

And yes the colors are now accessible.

Reply:
Also to be clear, the colors are now accessible but the sidebar itself is still not accessible for the reasons stated above. I don't have the time at the moment to find research an accessible solution, but for now, I believe the sidebar should not be included until it can be accessible. It's an enhancement at the moment so there is time to come up with a good solution.

@joelwass  Thoughts on waiting to implement the sidebar until an accessible solution is found?